gspread/worksheet.py

```diff
@@ -434,7 +446,15 @@ def get_values(self, range_name=None, **kwargs):
             worksheet.get_values('A2:B4', value_render_option=ValueRenderOption.formula)
         """
         try:
-            return fill_gaps(self.get(range_name, **kwargs))
+            vals = fill_gaps(self.get(range_name, **kwargs))
+            if combine_merged_cells:
```

Comment:
please compare to boolean value: `if combine_merged_cells is True`, otherwise someone can just pass any random object instead and it will pass the condition.